plotly/tools.py

```diff
@@ -48,11 +48,15 @@ def warning_on_one_line(message, category, filename, lineno, file=None, line=Non
 # this sets both the DEFAULTS and the TYPES for these items
 _FILE_CONTENT = {CREDENTIALS_FILE: {'username': '',
                                     'api_key': '',
+                                    'proxy_username': '',
+                                    'proxy_passwd': '',
+                                    'api_key': '',
                                     'stream_ids': []},
                  CONFIG_FILE: {'plotly_domain': 'https://plot.ly',
                                'plotly_streaming_domain': 'stream.plot.ly',
                                'plotly_api_domain': 'https://api.plot.ly',
-                               'plotly_ssl_verification': True}}
+                               'plotly_ssl_verification': True,
+                               'plotly_proxy_authorization': False}}
```

Comment:
This dev should work for both reverse and forward proxies. I'm tempted to keep it as `plotly_proxy_authorization`, though maybe I'm missing the source of potential confusion. Let me know! I can change it to `plotly_reverse_proxy_authorization` . nbd!